### PR TITLE
util.py fixes

### PR DESCRIPTION
**Pull request by pscheidler**
Fixed an error when using util -p
Added --min option to util.py
Fixed propagation of 'ids' option in toXml

### CODE REVIEW

File: ebmlite/util.py [hunk 1]
```diff
@@ -445,6 +445,9 @@ def errPrint(msg):
     argparser.add_argument('-p', '--pretty',
                            action="store_true",
                            help="Generate 'pretty' XML with ebml2xml.")
+    argparser.add_argument('--min',
+                           action="store_true",
+                           help="Generate minimal XML with ebml2xml. Just element nama and value")
```
Comment (StokesMIDE) [inline ebmlite/util.py:450]: typo

File: ebmlite/util.py [hunk 2]
```diff
@@ -470,10 +473,17 @@ def errPrint(msg):
...
     elif args.mode == "ebml2xml":
         doc = schema.load(args.input, headers=True)
+        if args.min:
+            root = toXml(doc, offsets=False, sizes=False, types=False, ids=False)
+        else:
         root = toXml(doc)  # , offsets, sizes, types, ids)
         s = ET.tostring(root, encoding="utf-8")
         if args.pretty:
-            parseString(s).writexml(out, addindent=b'\t', newl=b'\n', encoding=b'utf-8')
+            if 'b' in out.mode and out != sys.stdout:   # writexml wants to ooperate on a string, not binary output, file
```
Comment (StokesMIDE) [inline ebmlite/util.py:482]: typo

File: ebmlite/util.py [hunk 3]
```diff
@@ -470,10 +473,17 @@ def errPrint(msg):
         xml2ebml(args.input, out, schema)  # , sizeLength=4, headers=True, unknown=True)
     elif args.mode == "ebml2xml":
         doc = schema.load(args.input, headers=True)
+        if args.min:
+            root = toXml(doc, offsets=False, sizes=False, types=False, ids=False)
+        else:
         root = toXml(doc)  # , offsets, sizes, types, ids)
```
Comment (StokesMIDE) [inline ebmlite/util.py:479]: Indentation? Attempting to run the script fails here.

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
This PR currently fails the unit tests.

**Comment by StokesMIDE** [comment]
Thinking about it, users typically wouldn't want all that extra data, so omitting it would make more sense as the default. Maybe make the option `--max` instead? It's a minor change from what's there.

**Comment by pscheidler** [comment]
> Thinking about it, users typically wouldn't want all that extra data, so omitting it would make more sense as the default. Maybe make the option `--max` instead? It's a minor change from what's there.

Personally, I'd rather go with a solution that already exists and is backwards compatible

**Comment by StokesMIDE** [comment]
> Personally, I'd rather go with a solution that already exists and is backwards compatible

Yeah, I'm torn. Changing established behavior is generally something to avoid if possible, but on the other hand, the extra info is basically debugging stuff, and I'm not sure how many people would even notice if it were removed by default.

**Review by StokesMIDE — APPROVED** [review]
Looks good. 

It doesn't really change anything functionally, but ending with a newline is a PEP-8 recommendation. If you feel like adding it before merging, go for it; it won't affect my approval of the PR.

**Review by StokesMIDE — APPROVED** [review]

**Pull request closed without merging**